### PR TITLE
Change to single quote and .format()

### PR DESCRIPTION
**Pull request by fernandog**

### CODE REVIEW

File: sickbeard/postProcessor.py [hunk 1]
```diff
@@ -391,12 +392,13 @@ def _move(self, file_path, new_path, new_base_name, associated_files=False, subt
...
         def _int_move(cur_file_path, new_file_path):
 
-            self._log(u"Moving file from " + cur_file_path + " to " + new_file_path, logger.DEBUG)
+            self._log(u'Moving file from {0} to {1} '.format(cur_file_path, new_file_path), logger.DEBUG)
             try:
                 helpers.moveFile(cur_file_path, new_file_path)
                 helpers.chmodAsParent(new_file_path)
             except (IOError, OSError) as e:
-                self._log("Unable to move file " + cur_file_path + " to " + new_file_path + ": " + ex(e), logger.ERROR)
+                self._log('Unable to move file {0} to {1}: {!r}'.
```
Comment (p0psicles) [inline sickbeard/postProcessor.py]: should be {0} {1} {2!r}

File: sickbeard/postProcessor.py [hunk 2]
```diff
@@ -548,11 +550,11 @@ def _finalize(self, parse_result):
         else:
             logger.log(u"Parse result not sufficient (all following have to be set). will not save release name",
                        logger.DEBUG)
-            logger.log(u"Parse result(series_name): {}".format(parse_result.series_name), logger.DEBUG)
-            logger.log(u"Parse result(season_number): {}".format(parse_result.season_number), logger.DEBUG)
-            logger.log(u"Parse result(episode_numbers): {}".format(parse_result.episode_numbers), logger.DEBUG)
-            logger.log(u" or Parse result(air_date): {}".format(parse_result.air_date), logger.DEBUG)
-            logger.log(u"Parse result(release_group): {}".format(parse_result.release_group), logger.DEBUG)
+            logger.log(u"Parse result(series_name): {0}".format(parse_result.series_name), logger.DEBUG)
```
Comment (p0psicles) [inline sickbeard/postProcessor.py]: missed some double quotes

File: sickbeard/postProcessor.py [hunk 3]
```diff
@@ -577,7 +579,7 @@ def _analyze_name(self, name):
         try:
             parse_result = NameParser(True, tryIndexers=True).parse(name)
         except (InvalidNameException, InvalidShowException) as error:
-            logger.log(u"{}".format(error), logger.DEBUG)
+            logger.log(u"{0}".format(error), logger.DEBUG)
```
Comment (p0psicles) [inline sickbeard/postProcessor.py]: missed some double quotes

File: sickbeard/postProcessor.py [hunk 4]
```diff
@@ -803,32 +804,32 @@ def _get_quality(self, ep_obj):
...
-                u"Looking up quality for name " + cur_name + u", got " + common.Quality.qualityStrings[ep_quality],
+                u'Looking up quality for name {0}, got {1}'.format(cur_name, common.Quality.qualityStrings[ep_quality]),
                 logger.DEBUG)
 
             # if we find a good one then use it
             if ep_quality != common.Quality.UNKNOWN:
-                logger.log(cur_name + u" looks like it has quality " + common.Quality.qualityStrings[
-                    ep_quality] + ", using that", logger.DEBUG)
+                logger.log(u'{0} looks like it has quality {1}, using that'.format(cur_name, common.Quality.qualityStrings[
+                    ep_quality]), logger.DEBUG)
```
Comment (p0psicles) [inline sickbeard/postProcessor.py]: bad aligned. ep_qu.. should be below u'

File: sickbeard/postProcessor.py [hunk 5]
```diff
@@ -391,12 +392,13 @@ def _move(self, file_path, new_path, new_base_name, associated_files=False, subt
...
 
-            self._log(u"Moving file from " + cur_file_path + " to " + new_file_path, logger.DEBUG)
+            self._log(u'Moving file from {0} to {1} '.format(cur_file_path, new_file_path), logger.DEBUG)
             try:
                 helpers.moveFile(cur_file_path, new_file_path)
                 helpers.chmodAsParent(new_file_path)
             except (IOError, OSError) as e:
-                self._log("Unable to move file " + cur_file_path + " to " + new_file_path + ": " + ex(e), logger.ERROR)
+                self._log('Unable to move file {0} to {1}: {2!r}'.
+                            format(cur_file_path, new_file_path, e), logger.ERROR)
```
Comment (p0psicles) [inline sickbeard/postProcessor.py]: allign

File: sickbeard/postProcessor.py [hunk 6]
```diff
@@ -489,23 +491,23 @@ def _history_lookup(self):
...
+                names.append(self.nzb_name.rpartition('.')[0])
         if self.folder_name:
             names.append(self.folder_name)
 
         # search the database for a possible match and return immediately if we find one
         main_db_con = db.DBConnection()
         for curName in names:
-            search_name = re.sub(r"[\.\- ]", "_", curName)
+            search_name = re.sub(r'[\.\- ]', '_', curName)
             sql_results = main_db_con.select("SELECT showid, season, quality, version, resource FROM history WHERE resource LIKE ? AND (action % 100 = 4 OR action % 100 = 6)", [search_name])
```
Comment (p0psicles) [inline sickbeard/postProcessor.py]: line too long

File: sickbeard/postProcessor.py [hunk 7]
```diff
@@ -803,32 +804,32 @@ def _get_quality(self, ep_obj):
...
         ep_quality = common.Quality.assumeQuality(self.file_path)
         self._log(
-            u"Guessing quality for name " + self.file_name + u", got " + common.Quality.qualityStrings[ep_quality],
+            u'Guessing quality for name {0}, got {1}'.format(self.file_name, common.Quality.qualityStrings[ep_quality]),
             logger.DEBUG)
         if ep_quality != common.Quality.UNKNOWN:
-            logger.log(self.file_name + u" looks like it has quality " + common.Quality.qualityStrings[
-                ep_quality] + ", using that", logger.DEBUG)
+            logger.log(u'{0} looks like it has quality {1}, using that'
+                        .format(self.file_name, common.Quality.qualityStrings[ep_quality]), logger.DEBUG)
```
Comment (p0psicles) [inline sickbeard/postProcessor.py]: one space to the left

File: sickbeard/postProcessor.py [hunk 8]
```diff
@@ -910,7 +911,7 @@ def _is_priority(self, ep_obj, new_ep_quality):  # pylint: disable=too-many-retu
         if self.in_history or ep_obj.status in common.Quality.SNATCHED + common.Quality.SNATCHED_PROPER + common.Quality.SNATCHED_BEST:
             # if the episode is still in a snatched status, then we can assume we want this
             if not self.in_history:
-                self._log(u"Medusa snatched this episode and it is not processed before", logger.DEBUG)
+                self._log(u'Medusa snatched this episode and it is not processed before', logger.DEBUG)
                 return True
```
Comment (p0psicles) [inline sickbeard/postProcessor.py]: line 919 and down still allot of double quotes.

File: sickbeard/postProcessor.py [hunk 9]
```diff
@@ -979,22 +980,22 @@ def process(self):  # pylint: disable=too-many-return-statements, too-many-local
 
         # get the quality of the episode we're processing
         if quality and not common.Quality.qualityStrings[quality] == 'Unknown':
-            self._log(u"Snatch history had a quality in it, using that: " + common.Quality.qualityStrings[quality],
-                      logger.DEBUG)
+            self._log(u'Snatch history had a quality in it, using that: {0}'
+                        .format(common.Quality.qualityStrings[quality]), logger.DEBUG)
```
Comment (p0psicles) [inline sickbeard/postProcessor.py]: align, 1 space to left

File: sickbeard/postProcessor.py [hunk 10]
```diff
@@ -237,24 +237,25 @@ def recursive_glob(treeroot, pattern):
...
             # Add the extensions that the user doesn't allow to the 'extensions_to_delete' list
             if sickbeard.MOVE_ASSOCIATED_FILES:
-                allowed_extensions = sickbeard.ALLOWED_EXTENSIONS.split(",")
+                allowed_extensions = sickbeard.ALLOWED_EXTENSIONS.split(',')
                 found_extension = associated_file_path.rpartition('.')[2]
-                if not found_extension in allowed_extensions:
-                    self._log(u"Associated file extension not found in allowed extension {0}. Deleting it".format(found_extension), logger.WARNING)
+                if found_extension and not found_extension in allowed_extensions:
+                    self._log(u'Associated file extension not found in allowed extension: .{0}'
+                                .format(found_extension.upper()), logger.DEBUG)
```
Comment (p0psicles) [inline sickbeard/postProcessor.py]: align, one to left

Comment (fernandog) [inline sickbeard/postProcessor.py]: @p0psicles can't I used TAB? Do I have to manually delete back ?

Comment (p0psicles) [inline sickbeard/postProcessor.py]: If you have a proper IDE with pylint enabled, it will align it for you. Else you'll have to do it by hand yes.

**Comment by fernandog** [comment]
@p0psicles do we still use it?

`F: 29, 0: Unable to import 'adba' (import-error)`

File: sickbeard/postProcessor.py (hunk 10 above)
Comment (fernandog) [inline sickbeard/postProcessor.py]: my pylint doesnt complain about that

**Comment by fernandog** [comment]
rebased

**Comment by medariox** [comment]
Approved

**Pull request merged**